### PR TITLE
new persons in ESap011

### PR DESCRIPTION
**Pull request by eu-genia**

### CODE REVIEW

File: new/PRS12938Merrac.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1"><roleName type="title">ደጅአዝማች፡ </roleName>ምራጭ</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"><roleName type="title">dağğʾāzmāč</roleName> Mǝrrāč̣</persName>
+                    <persName xml:lang="gez" xml:id="n2" type="baptismal">ገብረ፡ እግዚአብሔር፡ </persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n2">Gabra ʾƎgziʾabǝḥer</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="political" cert="unknown">Could be the same as the early 20th-century
+                        governor of <placeName ref="LOC6145Wagari">Wagāriqo</placeName> (Liban, Eritrea).</occupation>
```
Comment (thea-m) [inline new/PRS12938Merrac.xml]: I don't know the workflow behind these files. Does the governor have an ID? Are IDs for the family members foreseen?
If the aim is to quickly create crucial references for the important entities mentioned in E-S records, ignore these comments.

File: new/PRS12937GabraWahedScribe.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12937GabraWahedScribe" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gabra Wahǝd</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS12937GabraWahedScribe.xml]: according to Pietro the titleStmt element is not printed anywhere and is not important, only persName is

File: new/PRS12938Merrac.xml (hunk 1 above)
Comment (eu-genia) [reply new/PRS12938Merrac.xml]: For the moment it is only for the E-S records, yes, there is no alternative ID for the governor,  this is the only Merrac, the identification is taken from the MS record

File: new/PRS12937GabraWahedScribe.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (thea-m) [reply new/PRS12937GabraWahedScribe.xml]: I disagree :) It is currently not printed anywhere and has no consequence for the view on the app, but I feel very strongly that our records should contain correct data.

[the remark this answers is not in the document]
Comment (eu-genia) [reply new/PRS12937GabraWahedScribe.xml]: done

**Pull request merged**